doc/manpages/bob-project.rst

```diff
@@ -167,6 +167,33 @@ to be passed on the command line *after* the package name.
     Add line to .config file. Can be used to specify preprocessor defines used by the QTCreator.
 
 
+Visual Studio Code project generator
+-----------------------------
```

Comment:
The line must be under the whole caption. Otherwise Sphinx will complain: `doc/manpages/bob-project.rst:171: WARNING: Title underline too short.`